prow/gerrit/util/util.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package util is a gerrit util library for common gerrit things
```

Comment:
https://blog.golang.org/package-names#TOC_5.

don't call packages util! at least call it `gerritutil` as a package name so you don't have to do that at the import site. imo this currently could be called `gerrit/projects` and the type be `s/ProjectsFlag/Flags` or something

Reply:
done